components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -110,11 +111,13 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
 
   closeEvent(): void {
     this.modifyID = false;
+    this.checkedProjectIDs = [];
```

Comment:
🎉 FYI: This is essentially emitting a checkedProjectIDs-list-has-changed event to the `ProjectsDropdownComponent`. Even though we are always setting it to an empty list it is still treated as a change event, which is good, because that is what we need for `ProjectsDropdownComponent` to reset its list of checked items to this "new" list of empty.
(For updating a team or token (in the *DetailComponents), `checkedProjectIDs` gets set to the starting value before the update so it does actually have a use. 😁 )